modules/rosa-create-objects.adoc

```diff
@@ -255,8 +260,11 @@ OVN-Kubernetes, the default network provider in ROSA 4.11 and later, uses the `1
 |--min-replicas <number_of_nodes>
 |Specifies the minimum number of compute nodes when enabling autoscaling. Default: `2`
 
+ifdef::openshift-rosa[]
+//this is being deprecated and will eventually be removed entirely.
 |--multi-az
 |Deploys to multiple data centers.
+endif::openshift-rosa[]
```

Comment:
Was told by Aaren that we should remove this flag entirely from HCP docs, since it is a no-op for HCP: https://redhat-internal.slack.com/archives/CB53T9ZHQ/p1752687583541179?thread_ts=1752674640.825629&cid=CB53T9ZHQ

Reply:
Thank you! I conditionalized the other instance too.